frontend/src/metabase/entities/data-apps/data-apps.ts

```diff
@@ -67,8 +72,8 @@ const DataApps = createEntity({
   },
 
   objectActions: {
-    scaffold: async ({ name, tables }: ScaffoldAppParams) => {
-      const dataApp = await DataAppsApi.scaffold({
+    scaffoldNewApp: async ({ name, tables }: ScaffoldNewAppParams) => {
+      const dataApp = await DataAppsApi.scaffoldNewApp({
```

Comment:
I wonder if it makes sense to DRY this up by making `scaffoldNewApp` call `scaffoldNewPages`? I say this because I suspect that this will undergo significant changes as we move to using models instead of tables. Also, it might make it easier to have a code path that doesn't start with a scaffold.

Reply:
Not sure I understand what you mean here. Do you mean like making a single action that would call `/api/app/:id/scaffold` if an app ID is passed and `/app/app/scaffold` if no ID is passed?